doc/changelog/06-Ltac2-language/20430-lconstr_in_ltac.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Enable use of (open\_)lconstr from Ltac2
```

Comment:
Just a nit, maybe you could write "inside Ltac2 Notation command"

Reply:
Adopted:)